app/uk/gov/hmrc/fileupload/write/envelope/Envelope.scala

```diff
@@ -17,25 +17,32 @@
 package uk.gov.hmrc.fileupload.write.envelope
 
 import cats.data.Xor
+import uk.gov.hmrc.fileupload.controllers.EnvelopeConstraints
 import uk.gov.hmrc.fileupload.write.envelope.Envelope.CanResult
 import uk.gov.hmrc.fileupload.write.infrastructure.{EventData, Handler}
 import uk.gov.hmrc.fileupload.{FileId, FileRefId}
 
 object Envelope extends Handler[EnvelopeCommand, Envelope] {
 
   type CanResult = Xor[EnvelopeCommandNotAccepted, Unit.type]
+  val defaultConstraints = EnvelopeConstraints(100, 25 * 1024 * 1024, 12 * 1024 * 1024)
```

Comment:
I don't mind if constraints are not read from config although they should be really, but maybe we could define them in one place as part of a static object and read them from there always, this gives one place to change them. This should be a very easy change